### PR TITLE
Update .gitignore and make clean. Fix namespace in install template.

### PR DESCRIPTION
**Pull request by jeffmendoza**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,16 +32,19 @@ bin/protoc-gen-docs*
 # Install generated files
 install/consul/istio.yaml
 install/eureka/istio.yaml
+install/kubernetes/addons/grafana.yaml
```
Comment (costinm) [inline .gitignore]: The right fix would be to not generate the files under the source tree, but under $OUT.

Can you make sure the files are deleted from git as well ? And I assume istio.VERSION may need to be ignored too ?

Comment (costinm) [reply .gitignore]: BTW - it's ok to leave the 'out of the source tree' for separate PR, but check if the files are not checked in, 
I think 2 are and 2 are not.

Comment (jeffmendoza) [reply .gitignore]: These files are not checked in, recently removed. Some updates had been committed to the generated files, then moved over to the template, causing some bugs.

Created #3496

istio.VERSION is currently checked in, but just gets updated. I'm not sure what the desired behavior is though.

**Pull request merged**